Lib/test/test_peepholer.py

```diff
@@ -463,6 +463,8 @@ def test_constant_folding(self):
             '(1, 2, -3)',
             '(1, 2, -3) * 6',
             'lambda x: x in {(3 * -5) + (-1 - 6), (1, -2, 3) * 2, None}',
+            '(1, )[0]',
+            '(1, (1, 2))[1][1]'
```

Comment:
Let's test also some expressions like `(1, 2-1, 3)[3-2]`. Also out of bounds cases like `(1,2)[6]` and `(1,2)[-1]`.